jekyll/_posts/2021-07-21-Nim-Efficienct-Expressive-Elecgant-Benchmarking.md

```diff
@@ -0,0 +1,369 @@
+| title | author | excerpt |
+| :---: | :----: | :---: |
+| Benchmarking the Beast | W. Gordon Goodsman (GordonBGood) | use CPU caches efficiently |
+
+Guest post
+This is a guest post by W. Gordon Goodsman (GordonBGood). If you would like to publish articles as a guest author on nim-lang.org then get in touch with us via Twitter or otherwise.
+
```

Comment:
You need to do it like this:

```
<div class="sidebarblock">
  <div class="content">
    <div class="title">Guest post</div>
    <div class="paragraph">
      This is a guest post by haxscramper.
      If you would like to publish articles as a guest author on nim-lang.org then get in touch with us via
      <a href="https://twitter.com/nim_lang">Twitter</a> or <a href="https://nim-lang.org/community.html">otherwise</a>.
    </div>
  </div>
</div>
```

Reply:
Okay, I'll make the change.